### DISCUSSION TITLE
Since updating to Quasar 2.16.0 functions returning the `any` type.

### DISCUSSION THREAD

**Discussion by MartinX3** [quasarframework/quasar#17170] [CLI - SSR mode] [answered]
I updated to Quasar 2.16.0 and @quasar/app-vite 2.0.0-beta.12.
Now eslint errors out because as example this variable has now the type `any`.
```typescript
import { date, } from 'quasar'
const daysInMonth = date.daysInMonth(new Date())
```

I look at the code in my IDE and also hover over the function.
It says the return type is `number`.
But eslint says `any`.
Adding the variable type via my IDE also puts the `any` at it.

And updating app-vite to beta 12 errors out with
```
Error [ERR_UNSUPPORTED_DIR_IMPORT]: Directory import '/home/runner/work/[...]/node_modules/quasar/wrappers' is not supported resolving ES modules imported from /home/runner/work/[...]/quasar.config.ts.temporary.compiled.1715059034987.mjs
Did you mean to import "quasar/wrappers/index.js"?
```

**Comment by UkonnRa** [comment]
Not only this, I received lots of `error TS7016: Could not find a declaration file for module 'quasar'. 'node_modules/quasar/dist/quasar.client.js' implicitly has an 'any' type` errors when linting.

`import { useQuasar } from "quasar"` is not working

**Comment by rstoenescu** [comment]
Hi @MartinX3 and @UkonnRa 

Can you put a reproduction repo here so I can take a look, pls? Also, what IDE (and version) are you using?

**Reply by rstoenescu** [reply, marked as answer]
Opened a ticket. Let's continue [there](https://github.com/quasarframework/quasar/issues/17172)
